keyboards/atreus/keymaps/khitsule/keymap.c

```diff
@@ -3,8 +3,10 @@
 #include QMK_KEYBOARD_H
 
 #define ALPH 0
-#define LOWR 1
-#define RAIS 2
+#define GAME 1
+#define LOWR 2
+#define RAIS 3
+#define ADJS 4
```

Comment:
We really do recommend an enum for the layers, as it's easier to manage long term. 

```suggestion
enum layers {
    ALPH,
    GAME,
    LOWR,
    RAIS,
    ADJS,
};
```

Reply:
Sure, thats fair enough and Ive made those changes. Probably worth noting that the defines method is used in the default keymaps so this will be a common PR suggestion